### PR TITLE
BETA: Register fajox.is-a.dev

### PR DESCRIPTION
**Pull request by FajoX1**
Added `fajox.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/fajox.json
Comment (Stef-00012) [inline domains/fajox.json:7]: I cannot reach your site (404 page)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/fajox.json
Comment (FajoX1) [reply domains/fajox.json:7]: it's github's "A" dns. fajox1.github.io

Comment (Stef-00012) [reply domains/fajox.json:7]: > it's github's "A" dns. fajox1.github.io. I will add it when will receive domain.

github pages uses 4 IPs for the A record, `185.199.108.153`, `185.199.109.153`, `185.199.110.153`, `185.199.111.153`

Comment (FajoX1) [reply domains/fajox.json:7]: > > it's github's "A" dns. fajox1.github.io. I will add it when will receive domain.
> 
> github pages uses 4 IPs for the A record, `185.199.108.153`, `185.199.109.153`, `185.199.110.153`, `185.199.111.153`

yes, can you give me domain and i will change dns to github's dns

Comment (wdhdev) [reply domains/fajox.json:7]: @FajoX1 You need to fix your domain first.

Comment (FajoX1) [reply domains/fajox.json]: > I cannot reach your site (404 page)

Now site work (fajox1.github.io) and i wrote dns, so, can you give me domain fajox.is-a.dev?

Comment (FajoX1) [reply domains/fajox.json]: @wdhdev

Comment (Stef-00012) [reply domains/fajox.json]: you have to wait for a maintainer to merge it now

**Review by satr14washere — APPROVED** [review]

**Pull request merged**
